External/Plugins/ProjectManager/Actions/FileActions.cs

```diff
@@ -445,34 +445,29 @@ public bool Rename(string oldPath, string newName)
 
                 if (isDirectory)
                 {
-                    // this is required for renaming directories, don't ask me why
-                    string oldPathFixed = (oldPath.EndsWith('\\')) ? oldPath : oldPath + "\\";
-                    string newPathFixed = (newPath.EndsWith('\\')) ? newPath : newPath + "\\";
-                    if (oldPathFixed.Equals(newPathFixed, StringComparison.OrdinalIgnoreCase))
```

Comment:
IIRC back in the day it was about normalizing the paths, because they could came from different sources and in different ways, so yeah, it could fail.